docs/diagrams/FindSequenceDiagram.puml

```diff
@@ -0,0 +1,55 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":FindCommandParser" as FindCommandParser LOGIC_COLOR
+participant "<<class>> \n ArgumentTokenizer" as ArgumentTokenizer LOGIC_COLOR
+participant "f:FindCommand" as FindCommand LOGIC_COLOR
+end box
+
+[-> LogicManager : execute("find 1")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand("find unpaid n/John s/<10")
+activate AddressBookParser
+
+create FindCommandParser
+AddressBookParser -> FindCommandParser
+activate FindCommandParser
+
+FindCommandParser -> AddressBookParser
```

Comment:
I think maybe you are missing a '-' here as the return arrow should be a dotted line?